nemo/collections/nlp/models/nlp_model.py

```diff
@@ -40,7 +39,6 @@
 from nemo.core.classes.exportable import Exportable
 from nemo.core.connectors.save_restore_connector import SaveRestoreConnector
 from nemo.utils import AppState, logging
-from nemo.utils.exp_manager import configure_checkpointing
```

Comment:
Why are these being removed? Not  needed?

Reply:
unused imports